test/Unit/Porter/Net/Http/HttpOptionsTest.php

```diff
@@ -65,6 +65,47 @@ public function testProxy()
         self::assertSame($host = 'https://foo.com:80', (new HttpOptions)->setProxy($host)->getProxy());
     }
 
+    public function testUserAgent()
+    {
+        self::assertSame($userAgent = 'Foo/Bar', (new HttpOptions)->setUserAgent($userAgent)->getUserAgent());
+    }
+
+    public function testFollowLocation()
+    {
+        $options = (new HttpOptions)->setFollowLocation(true);
+
+        self::assertTrue($options->getFollowLocation());
+        self::assertFalse($options->setFollowLocation(false)->getFollowLocation());
+    }
+
+    public function testRequestFullUri()
+    {
+        $options = (new HttpOptions)->setRequestFullUri(true);
+
+        self::assertTrue($options->getRequestFullUri());
+        self::assertFalse((new HttpOptions)->setRequestFullUri(false)->getRequestFullUri());
+    }
+
+    public function testMaxRedirects()
+    {
+        self::assertSame($maxRedirects = 10, (new HttpOptions)->setMaxRedirects($maxRedirects)->getMaxRedirects());
+    }
+
+    public function testProtocolVersion()
+    {
+        self::assertSame($protocolVersion = 1.1, (new HttpOptions)->setProtocolVersion($protocolVersion)->getProtocolVersion());
```

Comment:
Unfortunately our style checker does not detect long lines, but PSR-2 recommends a maximum line length of 120 characters. Since this exceeds the limit it must be broken down. Respecting other PSR-2 rules, it must therefore be formatted similar to the following.

```php
        self::assertSame(
                $protocolVersion = 1.1,
                (new HttpOptions)->setProtocolVersion($protocolVersion)->getProtocolVersion()
        );
```